Group_Constitution

```diff
@@ -0,0 +1,127 @@
+Constitution of “Mozilla Science Study Group- University of Toronto”
+
+1.	Name
+
+The official name of this recognized campus group is “     ”
+
+The acronym or abbreviation of this group is:      
+
+2.	Purpose and Objectives
+
+The purpose, objectives, mission and/or mandate of organization is outlined here:
+     
+
+3.	Membership
+
+Membership to the group is open to all the University of Toronto members (students, staff, faculty and alumni). 
+
+U of T members are permitted to run, nominate, and vote in elections and constitutional amendments. The group is open to non-U of T members. However, these members do not hold the aforementioned rights. Members must register with a designated executive by submitting their full name and a valid email address.
+
+The membership fee will be $      per year.
+
+Any member of the club may apply for a refund within one (1) month of becoming a member. No refunds are permitted 30 days after the election periods.
+
+For recognition by the University of Toronto Students’ Union (UTSU), the group must maintain a minimum of 30 members, a total of 51% of membership are UTSU members. The group must also maintain recognition from the Office of Student Life. These requirements are subject to change and should be checked with UTSU annually to ensure qualifications are met. 
+
+4.	Executive List and Duties
+
+The executive committee shall be comprised of three (3) elected officers. These include a President, Secretary, and Treasurer.
+
```

Comment:
@lwjohnst86:
This is the default template for group structure that ULife suggests. We can alter our executive positions and duties as we see fit! The membership fee is also something we can get rid of.